app/javascript/components/util/DateInput.js

```diff
@@ -54,7 +59,7 @@ class DateInput extends React.Component {
           key={this.props.tooltipKey || ''}
           placement={this.props.tooltipPlacement || 'auto'}
           overlay={<Tooltip style={this.props.tooltipText ? {} : { display: 'none' }}>{this.props.tooltipText}</Tooltip>}>
-          <div>
+          <div className={this.props.customClass?.includes('sm') ? 'date-input__datepicker-wrapper_sm' : ''}>
```

Comment:
I didn't make one for `md` and `lg` because they wouldn't be used. actually, this isn't even being used anymore because we got rid of the assessment/labs/histories filters so I'll remove this

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-78e7a05287c86c247421462e28de6baeadecd713623ec25ea9ae0a6b486a6896L62